iocage_lib/ioc_start.py

```diff
@@ -1312,19 +1311,49 @@ def __check_rtsold__(self):
             stdout=su.PIPE
         )
 
-    def get_default_gateway(self):
-        # e.g response - ('192.168.122.1', 'lagg0')
-        try:
-            return netifaces.gateways()["default"][netifaces.AF_INET]
-        except KeyError:
+    def get_default_interface(self):
+        if self.host_gateways['ipv4']['interface']:
+            return self.host_gateways['ipv4']['interface']
+        elif self.host_gateways['ipv6']['interface']:
+            return self.host_gateways['ipv6']['interface']
+        else:
             iocage_lib.ioc_common.logit(
                 {
                     'level': 'EXCEPTION',
-                    'message': 'No default gateway interface found'
+                    'message': 'No default interface found'
                 },
                 _callback=self.callback,
-                silent=self.silent
-            )
+                silent=self.silent)
+
+    def get_default_gateway(self, address_family='ipv4'):
+        gateway = self.host_gateways[address_family]['gateway']
+        interface = self.host_gateways[address_family]['interface']
+        if gateway and '%' in gateway:
+            interface_bridge_map = self.get_interface_bridge_map()
+            ipv6 = gateway.split('%')[0]
+            if interface in interface_bridge_map:
```

Comment:
@dlangille your issue seems to stem from this. As i do not have an ipv6 environment right now, would it be possible if you could test a patch for me ? I can bring up a PR and you can checkout to that branch and build iocage ?
Also @joachimmathes i may need to get in touch with you again to smooth this as we certainly are having regressions for ipv6 users making sure our new fixes don't regress for you. ( the issue @joachimmathes seems to stem from the fact that this change - `if gateway and '%' in gateway` - requires updates to the config which assume the user should know beforehand which bridge is going to be used. We most certainly do not want that as the idea is to make the process easier for the users. Btw can we please go through why we need to do this at all ? It seems it was working all right for @dlangille before and you yourself as well for other ipv6 use cases you have. Again apologies for a certain lack of knowledge here primarily because i can't test this myself right now ). Looking forward to hearing from you. Thank you!

Reply:
I only just saw this.  For those following along at home, we are athttps://github.com/iocage/iocage/pull/1016

Reply:
Thank you for your help on this. I have a working jail now.